src/components/Tooltip/AnnotationPopup.vue

```diff
@@ -257,18 +279,20 @@ export default {
           this.annotationEntry["featureId"]
         ) {
           const evidenceURLs = [];
-          this.evidence.forEach((evidence) => {
-            if (evidence.includes("DOI:")) {
-              const link = evidence.replace("DOI:", "https://doi.org/");
-              evidenceURLs.push(new URL(link));
-            } else if (evidence.includes("PMID:")) {
-              const link = evidence.replace(
+          this.evidence.forEach((evi) => {
+            let eviObject = {}
+            if (evi.includes("DOI:")) {
+              eviObject[evi] = evi.replace("DOI:", "https://doi.org/");
+            } else if (evi.includes("PMID:")) {
+              eviObject[evi] = evi.replace(
                 "PMID:",
                 "https://pubmed.ncbi.nlm.nih.gov/"
               );
-              evidenceURLs.push(new URL(link));
+            }
+            if (evi in eviObject) {
+              evidenceURLs.push(eviObject);
             } else {
-              evidenceURLs.push(evidence);
+              evidenceURLs.push(evi);
             }
           });
```

Comment:
This change here has broken my script as evidenceURLs has been modified to contain mixed type (string / object) and they are pushed onto the server.